openlibrary/catalog/get_ia.py

```diff
@@ -154,11 +155,10 @@ def get_from_archive_bulk(locator):
 
     assert 0 < length < MAX_MARC_LENGTH
 
-    ureq = urllib.request.Request(url, None, {'Range': 'bytes=%d-%d' % (r0, r1)})
-    f = urlopen_keep_trying(ureq)
+    f = urlopen_keep_trying(url, headers={'Range': 'bytes=%d-%d' % (r0, r1)}, stream=True)
     data = None
     if f:
-        data = f.read(MAX_MARC_LENGTH)
+        data = f.raw.read(MAX_MARC_LENGTH)
```

Comment:
I think `content` would be better here. `f.raw` is a urllib3.response.HTTPResponse object, and I can't get the `read()` to work in a way I'd expect.

Reply:
@hornc can you provide an example of a locator and a url please?   I would like to see what the data provided looks like so I can test this accurately.